Deploy/demos/devspaces/ttsa.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
To reuse the ttsa.yaml from helm folder as namespace is specified in the powershell command